UPDATING.md

```diff
@@ -63,7 +63,7 @@ https://developers.google.com/style/inclusive-documentation
 
 ### Deprecating ignore_first_depends_on_past on backfill command and default it to True
 
-When doing backfill with `depends_on_past` dags, users will need to pass `ignore_first_depends_on_past`.
+When doing backfill with `--depends-on-past` dags, users will need to pass `--ignore-first-depends-on-past`.
```

Comment:
```suggestion
When doing backfill with `depends_on_past` dags, users will need to pass `--ignore-first-depends-on-past`.
```

Reply:
`depends_on_past` is not a cli option. It is an arg you can pass to tasks (`BaseOperator`)

Reply:
Oh, yeah a mistake